axonserver/src/main/resources/static/settings.html

```diff
@@ -15,7 +15,7 @@
         SSL disabled
     </div>
 
-    <div v-if="node.developmentMode" id="developmentModeEnabled" class="msg warning">
+    <div v-if="node.developmentMode && !node.clustered" id="developmentModeEnabled" class="msg warning">
```

Comment:
I would use !isEnterprise() instead of !node.clustered (the isEnterprise() is available on all vue compents (defined in index.html). Same in line 100.